core/dbsecurity/scripts/sqcertgen

```diff
@@ -108,7 +108,7 @@ stateOrProvinceName_default     = California
 localityName                    = Locality Name (eg, city)
 localityName_default            = Cupertino
 0.organizationName              = Organization Name (eg, company)
-0.organizationName_default      = Hewlett-Packard Company
+0.organizationName_default      = Esgyn Corporation
```

Comment:
Will make the company name and location name generic since the customer needs to create the certificate file.